contracts/TokenRequest.sol

```diff
@@ -66,6 +87,31 @@ contract TokenRequest is AragonApp {
     */
     function setVault(address _vault) external auth(SET_VAULT_ROLE) {
         vault = _vault;
+        emit SetVault(_vault);
+    }
+
+    /**
+    * @notice Add `_token.symbol(): string` to the accepted deposit token request tokens
+    * @param _token token address
+    */
+    function addToken(address _token) external auth(MODIFY_TOKENS_ROLE) {
+        require(isContract(_token), ERROR_ADDRESS_NOT_CONTRACT);
+        require(!acceptedDepositTokens.contains(_token), ERROR_TOKEN_ALREADY_ACCEPTED);
+
+        acceptedDepositTokens.push(_token);
+        require(acceptedDepositTokens.length <= MAX_ACCEPTED_DEPOSIT_TOKENS, ERROR_TOO_MANY_ACCEPTED_TOKENS);
```

Comment:
this require should be avobe the acceptedDepositTokens.push right?

Reply:
We could put it above and change the `<=` to a `<`. If we put it above it as it is the max size of `acceptedDespositTokens` would be 101.

Reply:
ok is the same, feel free to merge it